site/en/docs/extensions/reference/permissions/index.md

```diff
@@ -56,21 +56,23 @@ Most Chrome extension permissions can be specified as optional, with the followi
     </tr>
     <tr id="debugger">
       <td><code>"debugger"</code></td>
-      <td>The <a href="debugger">chrome.debugger</a> API serves as an alternate transport for
-        Chrome's <a href="/devtools/docs/debugger-protocol">remote debugging protocol </a>.</td>
+      <td>The <a href="../debugger">chrome.debugger</a> API serves as an alternate transport for
+        Chrome's <a href="https://chromedevtools.github.io/devtools-protocol/">remote debugging
+        protocol</a>.</td>
     </tr>
     <tr id="declarativeNetRequest">
       <td><code>"declarativeNetRequest"</code></td>
-      <td>Grants the extension access to the <a
-          href="declarativeNetRequest">chrome.declarativeNetRequest</a> API.</td>
+      <td>Grants the extension access to the
+        <a href="../declarativeNetRequest">chrome.declarativeNetRequest</a> API.</td>
```

Comment:
```suggestion
        <a href="/docs/extensions/reference/declarativeNetRequest/">chrome.declarativeNetRequest</a>
        API.</td>
```

Generally we've prefered using the full absolute path in the extensions docs so that we can use find-replace to update all references in one go.